src/Nethermind/Nethermind.Evm.Test/EOFChecks/EOF4200Tests.cs

```diff
@@ -0,0 +1,304 @@
+                );
+
+                yield return ScenarioCase.CreateFromBytecode(
+                    BytecodeTypes.EvmObjectFormat,
+                    bytecode: Prepare.EvmCode
+                                .RJUMPV(new short[] { 1, 2, 4 }, 1)
+                                .INVALID()
+                                .INVALID()
+                                .PushData(2)
+                                .PushData(3)
+                                .MSTORE8(0, new byte[] { 1 })
+                                .RETURN(0, 1)
+                                .Done,
+                    databytes: Array.Empty<byte>(),
+                    expectedResults: (StatusCode.Success, null)
+                );
+
+                yield return ScenarioCase.CreateFromBytecode(
+                    BytecodeTypes.EvmObjectFormat,
+                    bytecode: Prepare.EvmCode
+                                .RJUMPV(new short[] { 1, 6 }, 0)
+                                .INVALID()
+                                .ADD(2, 3)
+                                .MSTORE8(0, new byte[] { 1 })
+                                .RETURN(0, 1)
+                                .Done,
+                    databytes: Array.Empty<byte>(),
+                    expectedResults: (StatusCode.Success, null)
+                );
+
+                yield return ScenarioCase.CreateFromBytecode(
+                    BytecodeTypes.EvmObjectFormat,
+                    bytecode: Prepare.EvmCode
+                                .RJUMPV(new short[] { 0, 5 }, 4)
+                                .ADD(2, 3)
+                                .MSTORE8(0, new byte[] { 1 })
+                                .RETURN(0, 1)
+                                .Done,
+                    databytes: Array.Empty<byte>(),
+                    expectedResults: (StatusCode.Success, null)
+                );
+
+                yield return ScenarioCase.CreateFromBytecode(
+                    BytecodeTypes.EvmObjectFormat,
+                    bytecode: Prepare.EvmCode
+                                .RJUMPV(new short[] { }, 0)
+                                .MSTORE8(0, new byte[] { 1 })
+                                .RETURN(0, 1)
+                                .Done,
+                    databytes: Array.Empty<byte>(),
+                    expectedResults: (StatusCode.Failure, "Scenario : Truncated Jumptable")
+                );
+
+                yield return ScenarioCase.CreateFromBytecode(
+                    BytecodeTypes.EvmObjectFormat,
+                    bytecode: Prepare.EvmCode
+                                .RJUMPV(new short[] { 1 }, 0)
+                                .MSTORE8(0, new byte[] { 1 })
+                                .RETURN(0, 1)
+                                .Done,
+                    databytes: Array.Empty<byte>(),
+                    expectedResults: (StatusCode.Failure, "Jump to Push Immediate")
```

Comment:
failure scenarios only for RJUMPV, no unit tests failure scenarios for RJUMP and RJUMPI